Library/Homebrew/formula_creator.rb

```diff
@@ -166,7 +166,7 @@ def install
             system "shards", "build", "--release"
             bin.install "bin/#{name}"
         <% elsif mode == :go %>
-            system "go", "build", *std_go_args
+            system "go", "build", *std_go_args(ldflags: "-s -w")
```

Comment:
should we consider making this the default `ldflags`?

Reply:
Only if it doesn't break any existing build. There was a concern that it could, but I don't have any examples.

Reply:
yeah, we can take granular steps to approach moving it into `std_go_args` :)